internal/tasks/manifests_test.go

```diff
@@ -755,3 +757,136 @@ func TestCheckVulnerabilitiesForNextManifestWithError(t *testing.T) {
 		`, image.Manifest.Digest, s.Clock.Now().Add(60*time.Minute).Unix(), s.Clock.Now().Unix(), clair.LowSeverity)
 	})
 }
+
+func TestCheckTrivySecurityStatusWithPolicies(t *testing.T) {
+	test.WithRoundTripper(func(_ *test.RoundTripper) {
+		j, s := setup(t, test.WithClairDouble, test.WithTrivyDouble)
+		tr, _ := easypg.NewTracker(t, s.DB.DbMap.Db)
+		trivyJob := j.CheckTrivySecurityStatus(s.Registry)
+
+		//upload an example image
+		image := test.GenerateImage(test.GenerateExampleLayer(4))
+		image.MustUpload(t, s, fooRepoRef, "latest")
+		tr.DBChanges().Ignore()
+		s.TrivyDouble.ReportFixtures[image.ImageRef(s, fooRepoRef)] = "fixtures/trivy/report-vulnerable-with-fixes.json"
+
+		//test baseline without policies
+		s.Clock.StepBy(1 * time.Hour)
+		expectSuccess(t, trivyJob.ProcessOne(s.Ctx))
+		expectError(t, sql.ErrNoRows.Error(), trivyJob.ProcessOne(s.Ctx))
+		tr.DBChanges().AssertEqualf(`
+			UPDATE blobs SET blocks_vuln_scanning = FALSE WHERE id = 1 AND account_name = 'test1' AND digest = '%[1]s';
+			UPDATE trivy_security_info SET vuln_status = '%[2]s', next_check_at = %[3]d, checked_at = %[4]d, check_duration_secs = 0 WHERE repo_id = 1 AND digest = '%[5]s';
+		`,
+			image.Layers[0].Digest,
+			clair.CriticalSeverity, s.Clock.Now().Add(60*time.Minute).Unix(), s.Clock.Now().Unix(), image.Manifest.Digest)
```

Comment:
I really don't like that go fmt is not strict about this new line here

Reply:
I was trying to avoid overlong lines, and matched the order of formatting directives.